zcash_client_backend/src/data_api/wallet/input_selection.rs

```diff
@@ -97,8 +99,8 @@ impl<FeeRuleT, NoteRef> Proposal<FeeRuleT, NoteRef> {
         &self.transparent_inputs
     }
     /// Returns the Sapling inputs that have been selected to fund the transaction.
-    pub fn sapling_inputs(&self) -> &[ReceivedSaplingNote<NoteRef>] {
-        &self.sapling_inputs
+    pub fn sapling_inputs(&self) -> Option<&SaplingInputs<NoteRef>> {
```

Comment:
This needs a changelogs entry.

Reply:
It's noted in the commit message, with the intent of being integrated into the changelog at release time as is proposed in #1032. I'm not sure yet how much I like this approach, but I think it's worth giving it a fair try for this release and then reassessing.
```
- `zcash_client_backend::data_api::wallet::input_selection::Proposal::sapling_inputs`
  now returns type `Option<&SaplingInputs>`.
```